examples/src/cube/mod.rs

```diff
@@ -5,13 +5,13 @@ use wgpu::util::DeviceExt;
 #[repr(C)]
 #[derive(Clone, Copy, Pod, Zeroable)]
 struct Vertex {
-    _pos: [f32; 4],
+    _pos: [f64; 4],
```

Comment:
To be resolved when no longer draft: Instead of hijacking cube, could this be a set of tests in `tests/tests`